packages/grafana-ui/src/components/ConfirmModal/ConfirmModal.tsx

```diff
@@ -52,7 +54,6 @@ export const ConfirmModal = ({
   body,
   description,
   confirmText,
-  confirmVariant = 'destructive',
```

Comment:
I removed it because I thought it was a duplicate of `confirmButtonVariant`, but it may need a review as I'm not 100% sure 🤷

Reply:
I can't find any reason to keep this but if you remove it in line 55 you also want to remove it from ConfirmModalProps (line 24).

Reply:
I investigated a bit more and discovered we are using this prop in some places. So, I'm going to undo this change in this PR and create a new one if I confirm this is not needed / duplicated.